jaxrs-spec/src/main/asciidoc/chapters/providers/_lifecycle_and_environment.adoc

```diff
@@ -33,6 +33,13 @@ discover provider classes via mechanisms such as class scanning. A
 JAX-RS implementation that supports automatic discovery of classes MUST
 process only those classes that are annotated with `@Provider`.
 
+[[extensions]]
+==== Extensions
```

Comment:
I would not call these `Extensions`. The name would be easily messed up with CDI Extensions, especially in 4.0. I believe these are `Services`.

Reply:
Good point, addressed in 30877eea03892c6fb5f53dee27a79ad9c7ce89f0